Projects/UOContent/Mobiles/Healers/BaseHealer.cs

```diff
@@ -14,8 +14,17 @@ public abstract partial class BaseHealer : BaseVendor
 
     private DateTime _nextResurrect;
 
-    public BaseHealer()
+    [SerializableField(0)] [SerializedCommandProperty(AccessLevel.GameMaster)]
```

Comment:
is this how this works? what do I need to do here?  this was how it was serialized in the `PricedHealer` before I moved his stuff here